src/components/IGVFileSelector.js

```diff
@@ -66,61 +68,83 @@ export class IGVFileSelector extends Component {
-                div({ style: { display: 'flex', alignItems: 'center' } }, [
-                  h(LabeledCheckbox, {
-                    checked: selectedFiles[name],
-                    onChange: () => this.toggleVisibility(name)
-                  })
-                ]),
-                h(Clickable, {
-                  style: styles.columnName,
-                  title: name,
-                  onClick: () => this.toggleVisibility(name)
-                }, [_.last(name.split('/'))])
-              ])
-            }
-          })
-        }
-      ]),
-      div({ style: { fontWeight: 500 } }, [
+      h(TitleBar, {
+        title: 'IGV',
+        onDismiss: clearSelectionsAndDismiss
+      }),
+      div({ style: { fontWeight: 500, paddingLeft: '1.25rem' } }, [
         'Reference genome: ',
-        div({ style: { display: 'inline-block', marginLeft: '0.25rem', minWidth: 125 } }, [
+        div({ style: { display: 'inline-block', marginLeft: '0.25rem', marginBottom: '1rem', minWidth: 125 } }, [
           h(Select, {
             options: ['hg38', 'hg19', 'hg18', 'mm10', 'panTro4', 'panPan2', 'susScr11',
               'bosTau8', 'canFam3', 'rn6', 'danRer10', 'dm6', 'sacCer3'],
             value: refGenome,
             onChange: ({ value }) => this.setState({ refGenome: value })
           })
         ])
-      ])
+      ]),
+      div({ style: Style.modalDrawer.content }, [
+        div({ style: { marginBottom: '1rem', display: 'flex' } }, [
+          div({ style: { fontWeight: 500 } }, ['Select:']),
+          h(Link, { style: { padding: '0 0.5rem' }, onClick: () => this.setAll(true) }, ['all']),
+          '|',
+          h(Link, { style: { padding: '0 0.5rem' }, onClick: () => this.setAll(false) }, ['none'])
+        ]),
+        h(AutoSizer, { style: { flex: 1 } }, [
+          ({ width, height }) => {
+            return h(List, {
+              height,
+              width,
+              rowCount: trackFiles.length,
+              rowHeight: 30,
+              noRowsRenderer: () => 'No valid files found',
+              rowRenderer: ({ index, style, key }) => {
+                const name = trackFiles[index]
+                return div({ key, index, style: { ...style, marginBottom: 'auto', display: 'flex' } }, [
+                  div({ style: { display: 'flex', alignItems: 'center' } }, [
+                    h(LabeledCheckbox, {
+                      checked: selectedFiles[name],
+                      onChange: () => this.toggleVisibility(name)
+                    })
+                  ]),
+                  h(Clickable, {
+                    style: styles.columnName,
+                    title: name,
+                    onClick: () => this.toggleVisibility(name)
+                  }, [_.last(name.split('/'))])
+                ])
+              }
+            })
+          }
+        ])
+      ]),
+      h(ButtonBar, {
+        style: Style.modalDrawer.buttonBar,
+        onDismiss: clearSelectionsAndDismiss,
```

Comment:
`ButtonBar` doesn't have an `onDismiss` prop, was this supposed to be `onCancel`?